src/windows/bindings.rs

```diff
@@ -0,0 +1,9866 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+// Bindings generated by `windows-bindgen` 0.49.0
+
+#![allow(
+    non_snake_case,
+    non_upper_case_globals,
+    non_camel_case_types,
+    dead_code,
+    clippy::all
+)]
+#[inline]
+pub unsafe fn ConvertSidToStringSidW<P0>(
+    sid: P0,
+    stringsid: *mut ::windows_core::PWSTR,
+) -> ::windows_core::Result<()>
+where
+    P0: ::windows_core::IntoParam<PSID>,
+{
+    ::windows_targets::link!("advapi32.dll" "system" fn ConvertSidToStringSidW(sid : PSID, stringsid : *mut ::windows_core::PWSTR) -> BOOL);
+    ConvertSidToStringSidW(sid.into_param().abi(), stringsid).ok()
```

Comment:
@kennykerr this is what I was talking about in https://github.com/microsoft/windows-rs/issues/2517, here and below `.ok()` is used and is unavailable